### ISSUE TITLE
AWS deployment does not seem to work?

### ISSUE THREAD

**Issue by wu4f** [OWASP/Serverless-Goat#14]
I just deployed Serverless-Goat on my AWS project at
https://eh2w2331i5.execute-api.us-east-1.amazonaws.com/Prod

I get green checkmarks that say everything has been deployed properly, but I get the following message on the API calls (rather than what I would normally get via https://www.serverless-hack.me/ )

Is there something I'm doing wrong or did something change in AWS?

{"message":"Forbidden"}

**Comment by cschnidr** [comment]
I had the same. After a while it started working...
Check also you bucket permissions if public access is blocked by default.

**Comment by wu4f** [comment]
I think I've tracked this down

The site that is brought up on AWS uses the "Prod" stage in the AWS Lambda.
https://eh2w2331i5.execute-api.us-east-1.amazonaws.com/Prod

The HTML does not take the stage into account.  When you try to convert the default URL, it goes to this endpoint (which doesn't exist and gives you the "Forbidden" message).
https://eh2w2331i5.execute-api.us-east-1.amazonaws.com/api/convert?document_url=https%3A%2F%2Fwww.puresec.io%2Fhubfs%2Fdocument.doc

Should the stage be added to the relative path of the form submission URL?
e.g. on Line 39 of src/frontend/index.html
  <form class="pure-form" action="Prod/api/convert">

**Comment by wu4f** [comment]
Duplicate of prior issue.   Fixed locally.
